lib/vm/src/errors.rs

```diff
@@ -55,6 +55,13 @@ pub enum Error {
         #[cfg(feature = "backtraces")]
         backtrace: snafu::Backtrace,
     },
+    // Similar to RuntimeErr but with custom message
+    #[snafu(display("Error during wasm execution: {}", msg))]
+    ExecutionErr {
```

Comment:
Okay, ExecutionErr is to cosmwasm-vm what ContractErr is to cosmwasm?
Seems good to have

Reply:
Yes, this is an error that happens inside of the VM, triggered by the contract.